src/opm/input/eclipse/EclipseState/SimulationConfig/SimulationConfig.cpp

```diff
@@ -113,6 +123,7 @@ namespace Opm {
         result.m_bcconfig = BCConfig::serializationTestObject();
         result.m_rock_config = RockConfig::serializationTestObject();
         result.m_useCPR = false;
+        result.m_useNONNC = false;
```

Comment:
this should be true - it should not be the default value as a missing serialization might slip through the cracks then (general rule, i of course see that you haven't forgotten it here..).

Reply:
ok :) I changed it to `result.m_useNONNC = true;`